webhooks-extension/docs/GettingStarted.md

```diff
@@ -0,0 +1,161 @@
+
+The [Tekton Dashboard](https://github.com/tektoncd/dashboard) is a general purpose, web-based UI for [Tekton Pipelines](https://github.com/tektoncd/pipeline). The Dashboard [Webhooks Extension](https://github.com/tektoncd/experimental/tree/master/webhooks-extension) allows users to set up GitHub webhooks that will trigger Tekton PipelineRuns and associated TaskRuns. This extension is intended to support Continuous Integration and Continuous Delivery (CI/CD) workflows. Git-driven workflow and automation is a common pattern that we expect most of our readers will be comfortable and familiar with. 
+
+This article aims to help you get webhooks up and running with Tekton. We talk about 'GitHub webhooks' in this article because as of August 2019, that is what the webhooks extension currently supports. Support for GitLab is on our roadmap but not yet implemented. 
+
+## Installation
+
+(Pointer to our main installation guide)
+
+## What are webhooks?
+
+As GitHub states, 
+> Webhooks allow external services to be notified when certain events happen. When the specified events happen, we’ll send a POST request to each of the URLs you provide. 
+
+So a 'webhook' is an outbound HTTP POST from GitHub to a given URL. It contains a rich payload in JSON format describing what just happened in Git. This might be, 'A Pull Request was opened', 'A commit was merged to a branch', or many other events. 
+
+It's important to understand that webhooks are outbound HTTP requests from GitHub to a given URL. This URL must be DNS-resolvable and IP-reachable from GitHub. If you are using `github.com` then the target URL must be on the external Internet. It is not generally possible to send a webhook from public `github.com` to a target (normally an Ingress endpoint on a Kubernetes cluster) behind a firewall. You should be using GitHub Enterprise today (or your own GitLab install in the future) if you wish to use webhooks to trigger Tekton Pipelines in a Kubernetes cluster situated behind a firewall. 
+
+## What does the Tekton Dashboard's Webhooks Extension do?
+
+The webhooks extension helps you to create a relationship between a git repository and a Tekton pipeline. This pipeline will be triggered when certain changes happen in the associated repository. Currently these changes are, 
+- A commit is merged into a branch, and
+- A Pull Request is created or modified
+
+See https://github.com/tektoncd/experimental/tree/master/webhooks-extension#notes-on-using-the-webhooks-extension for more details on how a webhook payload is turned into a `PipelineRun` and the necessary set of accompanying `PipelineResources`. 
+
+## Setting up a webhook from scratch
+
+Several steps must be completed in order to set up a working webhook. Pipeline definitions must be imported, and credentials set up to permit those pipelines to function. Credentials are typically required for GitHub, to create webhooks and clone repositories, and for a docker image registry to push images to. 
```

Comment:
Instead of `must be imported`, how about saying they must exist or be known on the cluster already? A user may think they've gotta import definitions but that's not really true, I could have applied them upfront ages ago